datashare-cli/src/main/java/org/icij/datashare/cli/DatashareCliOptions.java

```diff
@@ -789,6 +790,12 @@ public static void oauthClaimIdAttribute(OptionParser parser) {
                 .ofType(String.class);
     }
 
+    public static void searchQuery(OptionParser parser) {
+        parser.acceptsAll(singletonList(SEARCH_QUERY_OPT), "Json query for filtering index matches for EnqueueFromIndex task.")
```

Comment:
For the `searchQuery` option I would expect to have a lucene query string instead of a JSON body. Would be to have both (`searchQuery` and `searchBody`) but if I had to choose one I'll keep the first.

Reply:
It works with both I just added a test to show this. see 454b8333f9d2fc44542c2196c23e71210d6e44a0

Reply:
So cool! Thanks